### DISCUSSION TITLE
How to remove the buttons in the left operation bar

### DISCUSSION THREAD

**Discussion by alixiaodi** [ONLYOFFICE/DocumentServer#2336] [Q&A] [answered]
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/64963636/d7e02c16-f8af-4618-8e0a-cd3611211a51)

editorConfig: {
				mode: 'edit',
				lang: 'zh-CN',
				customization: {
					customer: {
						logo: 'http://127.0.0.1:8080/s/a3kvwg',
					},
				},
			},
I have used the above configuration but it does not take effect

Currently, this content cannot be modified or closed

**Comment by Rita-Bubnova** [comment]
@igwyd, Take a look, please.

**Comment by trofim24** [comment, marked as answer]
Hi, @alixiaodi 

You can customize/hide elements in the left operation bar using the white-label customization options. Take a look at the [documentation](https://api.onlyoffice.com/docs/docs-api/usage-api/config/editor/customization/customization-white-label/#about).

You can also try it out interactively in our [playground](https://api.onlyoffice.com/playground/?script=config&editor=word&mode=desktop&file=sample).

Please note: the parameters on this page are available only for the extended white label license of __[ONLYOFFICE Docs Developer](https://www.onlyoffice.com/developer-edition-prices.aspx?from=api)__. To request a quote, please contact our sales team at __[sales@onlyoffice.com](mailto:sales@onlyoffice.com)__.

Let us know if you have any further questions.
